async/cohttp_async.mli

```diff
@@ -20,18 +20,18 @@ open Async.Std
 
 (** Read in a full body and convert to a [string] *)
 
-module IO : Cohttp.S.IO with type 'a t = 'a Deferred.t
+module IO : (module type of Cohttp_async_io)
 
 module Request : sig
   type t = Cohttp.Request.t
   include Cohttp.S.Request with type t := Cohttp.Request.t
-  include Cohttp.S.Http_io with type t := Cohttp.Request.t and module IO=IO
```

Comment:
You're totally right. I'd originally removed the IO export (as that's what
the lwt version does) but then added it back when I realized that would be
a breaking change. I'll fix and amend.

On Tuesday, March 3, 2015, Rudi Grinberg <notifications@github.com> wrote:

> In async/cohttp_async.mli
> <https://github.com/mirage/ocaml-cohttp/pull/271#discussion_r25731132>:
>
> >
> >  module Request : sig
> >    type t = Cohttp.Request.t
> >    include Cohttp.S.Request with type t := Cohttp.Request.t
> > -  include Cohttp.S.Http_io with type t := Cohttp.Request.t and module IO=IO
>
> Why do you need this and the change below? Shouldn't changing module IO
> be enough?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mirage/ocaml-cohttp/pull/271/files#r25731132>.
>